packages/serum/src/market-proxy/middleware.ts

```diff
@@ -13,6 +13,8 @@ export interface Middleware {
   settleFunds(ix: TransactionInstruction): void;
   closeOpenOrders(ix: TransactionInstruction): void;
   prune(ix: TransactionInstruction): void;
+  consumeEvents(ix: TransactionInstruction): void;
```

Comment:
Want to do this in a separate PR? Since this should correspond with changes to https://github.com/project-serum/anchor/blob/master/spl/src/dex/middleware.rs#L44 .

Reply:
Or alternatively make the corresponding rust changes.

Reply:
I made rust changes. https://github.com/project-serum/anchor/pull/838